alacritty/src/input/keyboard.rs

```diff
@@ -644,3 +642,10 @@ impl From<ModifiersState> for SequenceModifiers {
         modifiers
     }
 }
+
+/// Check that the text represents either C0 or C1 codepoint.
+fn is_c0c1_codepoint(text: &str) -> bool {
+    let codepoint = text.chars().next().map(u32::from).unwrap_or(0);
+    // Include 0x7f as control character.
```

Comment:
This comment strikes me as odd because it doesn't document what the code does nor does it explain why `0x7f` is special and included.

Suggestion:

```
// 0x7f (DEL) is included here since it has a dedicated control code (`^?`) which generally does not match the reported text (`^H`), despite not technically being part of C0 or C1.
```

Not sure if this explanation matches your reasoning, but I'd assume it's something like that.